### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by tpolecat** [gfoot/sdcard6502#2]
Hi George,

Just wanted to say thank you for the effort you have put into this project. It's super super helpful, exactly what I was looking for! I have basically no idea what I'm doing and this was perfect for me. So, thanks!

rob

**Comment by scoco23** [comment]
Agreed.  This was extremely helpful.  If anyone has some resources for writing data to the SD card, that would be an excellent addition.

**Comment by liaminventions** [comment]
> Agreed. This was extremely helpful. If anyone has some resources for writing data to the SD card, that would be an excellent addition.

#5 in progress!
